torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2239,12 +2239,10 @@ options :class:`~torch.distributed.ProcessGroupNCCL.Options`).
               py::arg("timeout") = ::c10d::kProcessGroupNCCLDefaultTimeout,
               py::call_guard<py::gil_scoped_release>())
           .def(
-              "_abort",
```

Comment:
I am ok with this change but I am not sure if this is still used somewhere so it might break some use cases.. If it is used this PR might get reverted.

Reply:
I don't mind either, I just removed it based on @wanchaol's suggestion here: https://github.com/pytorch/pytorch/pull/111392#discussion_r1364583974

Reply:
Well, we do some use cases internally.

Reply:
This API was added by @pritamdamania87 a few weeks ago so I imagine the API usage should be relatively low and probably only @pritamdamania87 atm. We should remove it soon to avoid more ppl depend on it.